src/import-data/ImportData.ts

```diff
@@ -48,9 +48,12 @@ export const ImportData = () => {
 
   const { dataCatalog } = useDataCatalog();
   const snapshots = _.flow(
-    _.filter((snapshot: Dataset) => _.includes(snapshot['dct:identifier'] as string, snapshotIds)),
+    _.filter(
+      (snapshot: Dataset) => !!snapshot['dct:identifier'] && _.includes(snapshot['dct:identifier'], snapshotIds)
```

Comment:
Is `_.includes()` typed to require a defined argument? I would expect `_.includes(undefined, 'anything')` to return `false`.

Reply:
`_.includes(undefined, [undefined])` returns true.